docs/client/session.md

```diff
@@ -0,0 +1,24 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
+
+The primary purpose of sessions is to provide insight into the series of user actions or events that lead a critical or 
+interesting occurrence in the application's use. Sessions also provide a means to quantify overall application usage.
+
+
+## Attributes
+
+This attribute can be applied to all signals generated by an agent. It may be omitted from metrics, due to their aggregated nature. 
```

Comment:
I believe this was pointed out somewhere, but should be avoid the term agent when talking about instrumentation in general as not all client instrumentation will be done via agents?